firstvoices/backend/tests/test_apis/test_pages_api.py

```diff
@@ -6,15 +6,20 @@
 from backend.models.constants import AppRole, Role, Visibility
 from backend.models.widget import SiteWidget, SiteWidgetList, SiteWidgetListOrder
 from backend.tests import factories
-from backend.tests.test_apis.base_api_test import BaseControlledSiteContentApiTest
+from backend.tests.test_apis.base_api_test import (
+    BaseControlledSiteContentApiTest,
+    SiteContentPatchApiTestMixin,
+)
 from backend.tests.utils import (
     setup_widget_list,
     update_widget_list_order,
     update_widget_sites,
 )
 
 
-class TestSitePageEndpoint(BaseControlledSiteContentApiTest):
+class TestSitePageEndpoint(
+    BaseControlledSiteContentApiTest, SiteContentPatchApiTestMixin
```

Comment:
Tip: put the mixins first in the list so you know they will take priority and override the base class if there are any overlapping methods